README.md

```diff
@@ -1,34 +1,42 @@
 # Nextflow web site
 
-<https://nextflow.io> is a static web site. Pages are generated by using [JBake](http://jbake.org/).
+[nextflow.io](https://nextflow.io) is a static web site. Pages are generated by using [Astro](https://astro.build).
 
-Jbake configuration is controlled by file `jbake.properties`.
+## Project Structure
 
-## Content
+Inside of your Astro project, you'll see the following folders:
 
-Web site content is provided either with Markdown or HTML text file in the folder `content`.
-Each content file declares a `type` property that defines the template to be used.
-The following type are used :
+```text
+/
+├── public/
+└── src/
+    ├── content/
+    │   ├── blog/
+    │   └── podcast/
+    ├── components/
+    ├── layouts/
+    └── pages/
+```
 
+Blog posts and podcast episodes go into `content` as markdown files. Other top-level pages should be named as `.astro` or `.md` files in the `src/pages/` directory. Astro finds all of this content and builds a page. Each page is exposed as a route based on its file name.
 
-Page        | Type  | Template
------------ | ----- | -----------
-Normal      | page  | `page.ftl`
-Index       | index | `index.ftl`
-Blog        | -     | `archive.ftl` (`blog.html`)
-Podcasts    | -     | `podcasts.ftl` (`podcasts.html`)
-Blog posts  | post  | `post.ftl`
-Podcast     | post  | `podcast.ftl`
+There's nothing special about `src/components/`, but that's where we like to put any Astro/React/Vue/Svelte/Preact components.
 
+Any static assets, like images, can be placed in the `public/` directory.
 
-Template are contained in the folder `templates`. To add a custom template
-add an entry in the `jbake.properties` file as shown here: `template.<type>.file=<type>.ftl`.
+## Commands
 
-## Testing locally
+All commands are run from the root of the project, from a terminal:
 
-Generate the web site pages entering the command: `./jbake` and
-the resulting html are placed in the folder `output`.
+| Command                   | Action                                           |
+| :------------------------ | :----------------------------------------------- |
+| `npm install`             | Installs dependencies                            |
+| `npm run dev`             | Starts local dev server at `localhost:4321`      |
+| `npm run build`           | Build your production site to `./dist/`          |
+| `npm run preview`         | Preview your build locally, before deploying     |
+| `npm run astro ...`       | Run CLI commands like `astro add`, `astro check` |
+| `npm run astro -- --help` | Get help using the Astro CLI                     |
```

Comment:
We need a pipeline to try this out 😆

Reply:
Hah, yes it's a bit much. I'll replace with the `make` commands and hide these in a `<details>` dropdown 😉